src/machine-learning/machine-learning.ts

```diff
@@ -244,42 +246,53 @@ export class Model {
     this.client = client;
   }
 
+  /** The ID of the model. */
   get modelId(): string {
     return extractModelId(this.model.name);
   }
 
+  /** The model's display name. This is the value you use to refer
+      to the model in the iOS and Android libraries. */
   get displayName(): string {
     return this.model.displayName!;
   }
 
+  /** The model's tags. */
   get tags(): string[] {
     return this.model.tags || [];
   }
 
+  /** The date and time the model was created (added to the project). */
   get createTime(): string {
     return new Date(this.model.createTime).toUTCString();
   }
 
+  /** The date and time the model was last updated. */
   get updateTime(): string {
     return new Date(this.model.updateTime).toUTCString();
   }
 
+  /** The validation error message, if the model isn't valid. */
   get validationError(): string | undefined {
     return this.model.state?.validationError?.message;
   }
 
+  /** True if the model is published and available to download. */
   get published(): boolean {
     return this.model.state?.published || false;
   }
 
+  /** The model's ETag. */
   get etag(): string {
     return this.model.etag;
   }
 
+  /** The SHA256 hash of the model file. */
   get modelHash(): string | undefined {
     return this.model.modelHash;
   }
 
+  /** The model's tflite file. */
```

Comment:
Well, it's more like metadata about the model's tflite file. Where it was originally uploaded from or how it was entered into the system etc.